src/Resources/views/Admin/Page/_showInShopButton.html.twig

```diff
@@ -1,7 +1,7 @@
 {% set enabledChannels = resource.channels|filter(channel => channel.enabled == true) %}
 
 {% if sylius_bundle_loaded_checker('SyliusShopBundle') %}
-    {% if not resource.enabled or enabledChannels|length < 1 %}
+    {% if not resource.enabled or not resource.isPublished(date()) or enabledChannels|length < 1 %}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** Maybe we should keep the button even if the page is not published. This way you can check fast if your page is available on front or not.‏

Reply:
If the button is disabled, the page is not available too